src/ActorMultiVertex.cpp

```diff
@@ -389,7 +389,7 @@ CubicSplineN* ActorMultiVertex::GetSpline(size_t i)
 	return &(_splines[i]);
 }
 
-void ActorMultiVertex::SetState(size_t i)
+void ActorMultiVertex::SetState(int i)
```

Comment:
By fixing this warning, you may have caused a separate warning. I don't think int should compare against size_t.